src/bnb_trading/historical_tester.py

```
@@ -427,37 +471,39 @@
+                "long_signals": long_signals,
+                "short_signals": short_signals,
+                "long_accuracy": long_accuracy,
+                "short_accuracy": short_accuracy,
+                "overall_accuracy": overall_accuracy,
+                "total_pnl": self.baseline_metrics.total_pnl,
+                "max_drawdown": self.baseline_metrics.max_drawdown,
+                "sharpe_ratio": self.baseline_metrics.sharpe_ratio,
+                "avg_trade_duration": self.baseline_metrics.avg_trade_duration,
             }
 
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)